### PR TITLE
CODE RUB: action variable for testing exception in retrieve all in Assignment Attachement

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -28,9 +28,15 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllAssignmentAttachmentsWhen
                 broker.SelectAllAssignmentAttachments())
                     .Throws(sqlException);
 
-            // when . then
-            Assert.Throws<AssignmentAttachmentDependencyException>(() =>
-                this.assignmentAttachmentService.RetrieveAllAssignmentAttachments());
+            
+            // when
+            Action retrieveAllAssignmentAttachmentAction = () =>
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Exceptions.RetrieveAll.cs:33]: Attachments plural would be more appropriate

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
-﻿{
+{
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Properties/launchSettings.json:1]: This is specific to your local settings, do not check-in this.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Properties/launchSettings.json (hunk 2 above)
Comment (deepshri7) [reply OtripleS.Web.Api/Properties/launchSettings.json:1]: Can u explain me this.

Comment (ShriHumrudha) [reply OtripleS.Web.Api/Properties/launchSettings.json:1]: Revert the changes in this file.

Comment (deepshri7) [reply OtripleS.Web.Api/Properties/launchSettings.json:1]: Don't know how i messed up this.. I tried to fix

**Pull request closed without merging**